src/test/java/com/example/demo/SecurityIntegrationTest.java

```diff
@@ -0,0 +1,178 @@
+package com.example.demo;
+
+import com.example.demo.security.authInfo.AuthRequest;
+import com.example.demo.security.filter.JwtTokenFilter;
+import com.example.demo.security.service.UserService;
+import com.example.demo.security.userInfo.AppUser;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.MvcResult;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.setup.MockMvcBuilders;
+import org.springframework.web.context.WebApplicationContext;
+
+
+import java.util.ArrayList;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.springframework.security.test.web.servlet.setup.SecurityMockMvcConfigurers.springSecurity;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class SecurityIntegrationTest {
+    @Autowired
+    private WebApplicationContext webApplicationContext;
+
+    @Autowired
+    private UserService userService;
+
+    private MockMvc mvc;
```

Comment:
This field is autowired in your other test file like the MatcherControllerTest.java. Is there a reason why this one is not autowired in here?